tests/ca/new-build-cmd.sh

```diff
@@ -0,0 +1,5 @@
+source common.sh
+
+export NIX_TESTS_CA_BY_DEFAULT=1
+cd ..
+source ./build.sh
```

Comment:
This is us running the (remaining) `tests/build.sh` tests, except with CA derivations enabled by default. This shows that this functionality works with or with or without CA derivations.